.gitignore

```diff
@@ -37,4 +37,5 @@ package-lock.json
 .env.ody
 .env.desktop
 .env.lbrytv
+.env.defaults
```

Comment:
Defaults is needed in production build.  Use local override method mentioned above.